docs/api.md

```diff
@@ -137,6 +137,7 @@
   * [page.cookies([...urls])](#pagecookiesurls)
   * [page.coverage](#pagecoverage)
   * [page.deleteCookie(...cookies)](#pagedeletecookiecookies)
+  * [page.disableWebVitalsReporting()](#pagedisablewebvitalsreporting)
```

Comment:
This introduces a new pattern into Puppeteer for `.enableX()` and `.disableX()`. What’s the rationale behind this? (Is there a design doc?) Have we considered matching the `X.start()`/`X.stop()` precedent set by [the `Tracing` class](https://github.com/puppeteer/puppeteer/blob/v10.0.0/docs/api.md#class-tracing) instead?

Reply:
Following the pattern that we already established with the tracing is a great idea!